packages/shared/getElementFromTouchHitTarget.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {Fiber} from 'react-reconciler/src/ReactFiber';
+
+import {HostComponent} from 'shared/ReactWorkTags';
+import warning from 'shared/warning';
+
+export default function getElementFromTouchHitTarget(
+  targetFiber: Fiber,
+): null | Element {
```

Comment:
The return type here should be `Instance` from HostConfig since it's whatever the host config defined it to be. Which happens to be Element for DOM but you're calling it from noop too.